Can't build...

```
FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all dependencies for configuration ':compile'.
> Could not resolve :CoFHCore:[1.7.10]3.0.0B9-40-dev.
  Required by:
      org.sidoh:reactor_simulator:0.1.0
   > Could not GET 'https://libraries.minecraft.net//CoFHCore/%5B1.7.10%5D3.0.0B9-40-dev/CoFHCore-%5B1.7.10%5D3.0.0B9-40-dev.pom'. Received status code 403 from server: Forbidden
   > Could not GET 'http://addons.cursecdn.com/files/2221/835/CoFHCore-%5B1.7.10%5D3.0.0B9-40-dev.xml'.
      > org.apache.http.client.ClientProtocolException (no error message)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 2 mins 2.996 secs

```

--- a/build.gradle
+++ b/build.gradle
@@ -51,7 +51,7 @@ repositories {
 
 dependencies {
     compile 'com.mod-buildcraft:buildcraft:6.0.18:dev'
-    compile 'li.cil.oc:OpenComputers:MC1.7.10-1.4.0.+:api'
+    compile 'li.cil.oc:OpenComputers:MC1.7.10-1.4.+:api'
     compile name: 'CoFHCore', version: '[1.7.10]3.0.0RC7-211-dev', ext: 'jar'
 
     // :'(

--- a/build.gradle
+++ b/build.gradle
@@ -27,8 +27,8 @@ archivesBaseName = "reactor_simulator"
 mainClassName = 'org.sidoh.reactor_simulator.service.SimulatorServer'
 
 minecraft {
-    version = "1.7.10-10.13.2.1232"
-    runDir = "run"	
+    version = "1.7.10-10.13.2.1291"
+    runDir = "run"
 
     replaceIn "BRConfig.java"
     replace "@VERSION@", project.version
@@ -45,14 +45,14 @@ repositories {
     }
     ivy {
         name "CoFHCore"
-        artifactPattern "http://addons.cursecdn.com/files/2221/835/[module]-[revision].[ext]"
+        artifactPattern "http://addons-origin.cursecdn.com/files/2230/211/[module]-[revision].[ext]"
     }
 }
 
 dependencies {
     compile 'com.mod-buildcraft:buildcraft:6.0.18:dev'
     compile 'li.cil.oc:OpenComputers:MC1.7.10-1.4.0.+:api'
-    compile name: 'CoFHCore', version: '[1.7.10]3.0.0B9-40-dev', ext: 'jar'
+    compile name: 'CoFHCore', version: '[1.7.10]3.0.0RC7-211-dev', ext: 'jar'
 
     // :'(
     compile fileTree(dir: 'lib', include: ['*.jar'])
@@ -74,11 +74,11 @@ processResources
     // replace stuff in mcmod.info, nothing else
     from(sourceSets.main.resources.srcDirs) {
         include 'mcmod.info'
-                
+
         // replace version and mcversion
         expand 'version':project.version, 'mcversion':project.minecraft.version
     }
-        
+
     // copy everything else, thats not the mcmod.info
     from(sourceSets.main.resources.srcDirs) {
         exclude 'mcmod.info'

